src/Apps/Order/OrderApp.tsx

```diff
@@ -80,7 +78,10 @@ export class OrderApp extends React.Component<OrderAppProps, OrderAppState> {
   }
 
   render() {
-    const { children } = this.props
+    const { children, location, router, order, params } = this.props
+    if (order.state !== "pending" && !location.pathname.includes("status")) {
+      router.replace(`/order2/${params.orderID}/status`)
+    }
```

Comment:
@zephraph This seems to be causing an error: `TypeError: protocol.transition is not a function` both in staging and locally. Is this possibly because this component is SSR-ed but some variables are missing?

```
(apps/order2) Error:  TypeError: protocol.transition is not a function
    at /Users/yuki/GitHub/force/node_modules/farce/lib/createHistoryMiddleware.js:35:49
    at /Users/yuki/GitHub/force/node_modules/farce/lib/createLocationMiddleware.js:26:20
    at /Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:154:22
    at /Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:75:12
    at /Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:61:29
    at runHook (/Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:43:12)
    at runHooks (/Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:60:10)
    at runAllowTransition (/Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:74:10)
    at /Users/yuki/GitHub/force/node_modules/farce/lib/createTransitionHookMiddleware.js:144:20
    at /Users/yuki/GitHub/force/node_modules/farce/lib/ensureLocationMiddleware.js:33:18
    at Object.replace (/Users/yuki/GitHub/force/node_modules/found/node_modules/redux/lib/redux.js:455:12)
    at OrderApp.replace [as render] (/Users/yuki/GitHub/force/node_modules/@artsy/reaction/src/Apps/Order/OrderApp.tsx:83:14)
    at processChild (/Users/yuki/GitHub/force/node_modules/react-dom/cjs/react-dom-server.node.development.js:2478:18)
    at resolve (/Users/yuki/GitHub/force/node_modules/react-dom/cjs/react-dom-server.node.development.js:2335:5)
    at ReactDOMServerRenderer.render (/Users/yuki/GitHub/force/node_modules/react-dom/cjs/react-dom-server.node.development.js:2657:22)
    at ReactDOMServerRenderer.read (/Users/yuki/GitHub/force/node_modules/react-dom/cjs/react-dom-server.node.development.js:2631:23)
    at Object.renderToString (/Users/yuki/GitHub/force/node_modules/react-dom/cjs/react-dom-server.node.development.js:3022:25)
    at renderToString (/Users/yuki/GitHub/force/node_modules/@artsy/reaction/src/Artsy/Router/buildServerApp.tsx:72:22)
    at tryCatch (/Users/yuki/GitHub/force/node_modules/regenerator-runtime/runtime.js:62:40)
    at Generator.invoke [as _invoke] (/Users/yuki/GitHub/force/node_modules/regenerator-runtime/runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/yuki/GitHub/force/node_modules/regenerator-runtime/runtime.js:114:21)
    at asyncGeneratorStep (/Users/yuki/GitHub/force/node_modules/@artsy/reaction/dist/Artsy/Router/buildServerApp.js:34:103)
    at _next (/Users/yuki/GitHub/force/node_modules/@artsy/reaction/dist/Artsy/Router/buildServerApp.js:36:194)
    at run (/Users/yuki/GitHub/force/node_modules/core-js/modules/es6.promise.js:75:22)
    at /Users/yuki/GitHub/force/node_modules/core-js/modules/es6.promise.js:92:30
    at flush (/Users/yuki/GitHub/force/node_modules/core-js/modules/_microtask.js:18:9)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
```

Reply:
Oh... yeah, you might be right. I bet `router.replace` doesn't work on the server.